Mathlib/Analysis/Convex/Continuous.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2023 Yaël Dillies, Zichen Wang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Zichen Wang
+-/
+import Mathlib.Analysis.Convex.Normed
+
+/-!
+# Convex functions are continuous
+
+This file proves that a convex function from a finite dimensional real normed space to `ℝ` is
+continuous.
+-/
+
+open FiniteDimensional Metric Set List Bornology
+open scoped Topology
+
+variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
+  {C : Set E} {f : E → ℝ} {x₀ : E} {ε r : ℝ}
+
+lemma ConvexOn.exists_lipschitzOnWith_of_isBounded (hf : ConvexOn ℝ (ball x₀ r) f) (hε : 0 < ε)
+    (hf' : IsBounded (f '' ball x₀ r)) : ∃ K, LipschitzOnWith K f (ball x₀ (r - ε)) := by
```

Comment:
How about:
```suggestion
lemma ConvexOn.exists_lipschitzOnWith_of_isBounded (hf : ConvexOn ℝ (ball x₀ r) f)
    (hf' : IsBounded (f '' ball x₀ r)) {r' : ℝ} (h : r' < r) :
    ∃ K, LipschitzOnWith K f (ball x₀ r') := by
```

Reply:
I am not convinced. As you see in the calculations, `ε` is the important quantity, and one could write a version of the lemma giving the Lipschitz constant in terms of the bound and `ε`.